subcommittees/mentoring/roles/mentoring-coordinator.md

```diff
@@ -0,0 +1,24 @@
+# Mentoring Coordinator
+
+## Responsibilities
+
+- Recruit mentors and mentees for the Mentoring Program.  
+- Pair Mentors and Mentees based on responses to Google form surveys.  
+- Set up virtual meeting rooms and Etherpads for each mentoring team.  
+- Send email introductions for each mentoring team.  
+- Follow up with each mentoring team on a monthly basis and ask if there's anything you can do to help.  
+- Keep track of program completion rates to assess success.  
+- Get feedback from Mentors and Mentees about program.  
+
+## Time commitment
```

Comment:
This section and the following few are empty.  I think we should fill them in or drop the headers.